pages/How-to/guide/adding-issue-template.md

```diff
@@ -1,3 +1,114 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+# How to create an issue?
+
+-> Issues can be created in a variety of ways.
+
+-> Issues can be used to keep track of bugs, enhancements, or other requests.
+
+<br/>
+
+## Creating an issue in open-source-with-pradumna.
+<br/>
+
+-> On GitHub.com, navigate to the main page of the repository.
+
+<br/>
+
+![issue2](../images/issue2.png)
+
+<br/>
+
+-> Under your repository name, click on `New Issue`
+
+<br/>
+
+
+
+![issue7](../images/issue7.png)
+
+
+<br/>
+
+
+-> When you click on `New Issue` , you will get a list of 4 types of issues.
+
+<br/>
+
+![issue1](../images/issue1.png)
+
+
+<br/>
+
+## Let's check all of them one by one
+
+</br>
+
+### 🐛 Bug
+
+
+</br>
+
+-> If you find any bug in the repository feel free to click on `Get Started` and you will get a template in which you will get a template in which you will write a crisp title of the bug and then in the description, ellaborate the bug.
```

Comment:
Hey @Kanika637 there is a typo at the last of this line of code. ( Replace ellaborate with elaborate )